### DISCUSSION TITLE
The Thingy

### DISCUSSION THREAD

**Discussion by Revnova** [nzp-team/nzportable#894] [Map Releases]
# Still a W.I.P.
Currently frames drop when you're inside the hangar looking back at spawn. This map does not work on anything other than FTE (PC).

# [Download Here (day)](https://github.com/user-attachments/files/18112211/those-damned-norwegians.zip)
# [Download Here (night)](https://github.com/user-attachments/files/18112232/those-damned-norwegians_night.zip)

its a snow map inspired by John Carpenter's The Thing. but this is my take on it. its not meant to replicate it.
Currently has:
-Random Perk Machines
- Teddies
- Power Switch
- Mystery Box/TP Spots
- EE Song
- Unlockables
- PaP
- Teleporter
- Buyable ending

# Screenshots
![image](https://github.com/user-attachments/assets/8566abf7-53c4-4cbd-a5b6-cbb73f7ab59b)
![image](https://github.com/user-attachments/assets/e33c83c2-c276-48da-9817-6ca410634374)
![image](https://github.com/user-attachments/assets/22c3ba9a-56b5-48af-bc6e-b532815f8975)
![image](https://github.com/user-attachments/assets/a80eb775-831f-401b-b9ea-3c22fe84ceb4)

day:
![image](https://github.com/user-attachments/assets/e8c9723a-deb2-4a3c-81d0-c029981c1bba)
![image](https://github.com/user-attachments/assets/aad8276e-c31b-4476-8620-ce93417c0894)
![image](https://github.com/user-attachments/assets/b36226ad-eb4b-41ac-b588-5184ff1cd32f)

night:
![image](https://github.com/user-attachments/assets/860a826a-bf23-4cc2-b773-623e352770ae)
![image](https://github.com/user-attachments/assets/cdba489e-41c2-4f37-946c-d3a02afa9c3e)
![image](https://github.com/user-attachments/assets/69478c8e-4bad-4afa-a37c-aa46c7229672)

**Comment by Revnova** [comment]
Updated:
-added more spawns for indoor areas
-more teddies
-fixed textures in certain areas.
-finished waypoints
-added EE Song (thanks to Jacob Giguere)

**Comment by Revnova** [comment]
Updated:
Ported hlmdl to qmdl. but you still get an error on anything other than FTE(PC)

**Comment by Revnova** [comment]
Updated:
-added Teleporter
-added PaP

**Comment by tottalynotevil34332** [comment]
hey man this looks cool as shit cant wait to play

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as its FTE only and is not uploaded to archive.org

**Comment by Revnova** [comment]
# Updated!
- reset price of perks.
- set QR to 1500 in solo
- fixed first Bar being inaccessible
- doors prices have been changed since the last time i posted about an update
- upped the requirement of teddies by 2
- added light trail
- change the path of models from "converted" to "revs" (if you have a folder named converted in your models directory. you should delete it)
- made a night version of the map with its own download
- there's now a buyable ending

**Comment by Jeyphr** [comment]
Check out my review here: https://youtu.be/1aTpbMJqPAg

**Reply by maximhelper** [reply]
@Jeyphr Do you have a discord by any chance? I would like to join. I'm Max or Maxim BTW, you played with me the other day.

**Comment by scootsterling** [comment]
Downloaded to try on PS Vita despite it only working on PC, Day map has a fog but seems to work. Night map gave me a red screen but turning fullbright on got rid of it and made it look like the daytime map without the fog, didn't really play either yet, just tested them for a few minutes and bought a few doors, seems to work though.

**Comment by Spacecube-9312** [comment]
I just wish that the map should utilize more from the source material “The Thing” as the facility should be shrouded in mystery to what have happened.
But never watched the movie, so I can’t really make a map of my own take on this concept
